packages/react/src/components/Cta/Cta.js

```diff
@@ -22,46 +22,56 @@ import React from 'react';
  * @param {object} props.cta cta object which includes url, link text and target properties.
  * @returns {*} CTA Component
  */
-const CTA = ({ style, cta }) =>
+const CTA = ({ style, type, ...cta }) =>
```

Comment:
One thing to note here is the style, type and even cta props are at the same level now. We basically merged style and type into whatever the style type's data model is. For instance...

LinkWithIcon
```js
{
  style: 'text', // not apart of the original LinkWithIcon data model
  type: 'jump', // not apart of the original LinkWithIcon data model
  href: '...', // apart of the original LinkWithIcon data model
  copy: '...' // apart of the original LinkWithIcon data model
  // icon added via CTA component based on type
  // target, or onclick is determined by CTA component
}
```